modules/st2-login/login.component.js

```diff
@@ -207,8 +207,8 @@ export default class Login extends React.Component {
             <a target="_blank" rel="noopener noreferrer" href="http://docs.stackstorm.com">
               Documentation
             </a>
-            <a href="mailto:support@stackstorm.com" data-reamaze-lightbox="contact">
```

Comment:
It's funny how the PR title advertizing "Replace Reamaze integration", but in fact removes also Support Email.